Dockerfile

```diff
@@ -10,7 +10,7 @@ FROM python:3.9-alpine as base
 COPY --from=rust_builder /usr/local/cargo/bin/wit-bindgen /usr/bin
 COPY --from=rust_builder /writ /writ
 COPY --from=rust_builder /wasmtime-py /wasmtime-py
-RUN apk add libgcc libc6-compat && \
+RUN apk add libgcc libc6-compat gdb && \
```

Comment:
Just noticed this -- up on line 6, there's a `git checkout fix-usage` hard-coded in.  This was a hack and needs to be removed.